CHANGELOG.md

```diff
@@ -2,7 +2,14 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Upgrade to v3.0.0 of the AWS Terraform Provider
+  **Please Note:**
+  * A number of `deprecated` properties on resources have been removed. You can check the full list in the 
+  [upstream CHANGELOG](https://github.com/pulumi/terraform-provider-aws/blob/upstream-v3.0.0/CHANGELOG.md#300-july-31-2020)
+  * v3.0.0 introduces:
+    * Improved Credential Ordering
+    * Major changes to the `aws.acm.Certificate` to fix a number of issues regarding perpetual diffs of propertes
+    * Enabling drift detection of `loadBalancers` and `targetGroupArns` in `aws.autoscaling.Group`
```

Comment:
Can we structure this to make more clear that the full set of changes are in the upstream CHANGELOG - but that some of the most notable changes are listed explicitly here?  Currently it reads like there are some deprecated properties removed and then three other changes - which I don't think is true?